buddyboss/buddyboss-platform
# Fixed - Duplicate New Reply Notification

### All Submissions:






### Changes proposed in this Pull Request:
I have updated the reply author id instead of the topic author id to fix multiple notifications.


Fixes #1880 .

### How to test the changes in this Pull Request:

1. Create a new discussion
2. Reply to the discussion using different account ex: Jessica (The creator of the discussion will receive 1 new reply notification)
3. Again, switch to another user account ex: Arianna
4. Reply to Jessica reply
5. Check the discussion creator notification. Only 1 notification should be display now.

### Proof Screenshots or Video

https://www.loom.com/share/4913ebaf072c4b77bc9909fc235608f3


### Other information:
